src/main/java/com/powsybl/openloadflow/ac/AcOuterLoopContext.java

```diff
@@ -18,12 +18,23 @@
  */
 public class AcOuterLoopContext extends AbstractOuterLoopContext<AcVariableType, AcEquationType, AcLoadFlowParameters, AcLoadFlowContext> {
 
+    private int iteration; // current iterations of this single outer loop type
+
     private AcSolverResult lastSolverResult;
 
     AcOuterLoopContext(LfNetwork network) {
         super(network);
     }
 
+    @Override
+    public int getIteration() {
```

Comment:
Do not override the getters/setters. This is managed by the base class now.

Reply:
same, fixed in [269b938](https://github.com/powsybl/powsybl-open-loadflow/pull/1123/commits/269b93899538fd33f1e73cb50ffdbf2bcc87082e)